### DISCUSSION TITLE
Is possible to use Zodis outside a monorepo?

### DISCUSSION THREAD

**Discussion by rodolphoasb** [ecyrbe/zodios#328] [Q&A]
I found out that working with React Native in a monorepo is a nightmare!

So I'm planning to have a separate project only for React Native and use Nextjs API's as my backend (on another project). Can we share API type definitions across the two separate projects?

**Comment by ecyrbe** [comment]
Yes you can share it if you put it on a library. But you'll loose Dynamic editing feature demonstrated on video..
